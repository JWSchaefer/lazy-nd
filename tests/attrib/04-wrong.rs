use lazy_nd::lazy_nd;

#[lazy_nd(lim = D)]
struct Test<const D: usize> {}
fn main() {}
