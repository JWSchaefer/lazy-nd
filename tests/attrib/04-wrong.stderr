error: Unexpected token `lim`.
 --> $DIR/04-wrong.rs:3:11
  |
3 | #[lazy_nd(lim = D)]
  |           ^^^
